_data/sites.json

```diff
@@ -3046,6 +3046,16 @@
         ]
     },
 
+    {
+        "name": "Habitica",
+        "url": "https://habitica.com/user/settings/site",
+        "difficulty": "easy",
+        "notes": "Login to your account, click the User icon in the top-right corner, click on Settings in the drop-down menu, and click on the red Delete Account button at the bottom of the page.",
```

Comment:
A little nit-picky, but "Login to your account" should be "Log in to your account".